page_creation.rst

```diff
@@ -104,7 +104,7 @@ You can now add your route directly *above* the controller:
         // src/Controller/LuckyController.php
 
         // ...
-        + use Symfony\Component\Routing\Annotation\Route;
+        use Symfony\Component\Routing\Annotation\Route;
```

Comment:
we also need to remove the plus character on lines 111 to 113 as well as on lines 125 and 129

Reply:
I removed the extra `+` while merging. Thanks!